test/tests.Spec.js

```diff
@@ -0,0 +1,154 @@
+'use strict';
+
+var chai = require('chai');
+var expect = chai.expect;
+var fs = require('fs-extra');
+var proxyquire = require('proxyquire');
+var sinon = require('sinon');
+
+
+describe('Given the KarmaCoveralls Module', function () {
+
+  var karmaCoveralls, coverallsMock, dir, file;
+
+  before(function (done) {
+
+    dir = 'tmp/';
+    file = 'lcov.info';
+
+    coverallsMock = {
+      getBaseOptions: function(fn) {
+        fn(false, {filepath: ''})
+      },
+      convertLcovToCoveralls: function(input, options, cb) {
+        cb();
+      },
+      sendToCoveralls: function(postData, cb) {
+        cb()
+      }
+    };
+
+    sinon.spy(coverallsMock, 'sendToCoveralls');
+
+    karmaCoveralls = proxyquire('../lib/index.js', { 'coveralls': coverallsMock });
+
+
+    // Creates a fake code coverage report file
+    fs.mkdirs(dir, function (err) {
```

Comment:
maybe there's a better way --- how about just adding a fixture file that stays in the tree?